src/commands/admin/set.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
     const item =
       message.mentions.channels.first() || message.mentions.roles.first();
 
-    if (!option) return message.channel.send("Please provide an valid option");
+    if (!option) return message.channel.send("Please provide an valid option (\`suggest-channel / announce-channel / welcome-channel / leave-channel / audit-channel / welcome-role / mod-log\`)");
```

Comment:
Could you please format this as (`suggest-channel`, `announce-channel`, ...). Makes it easier for a user to copy it.

Reply:
Sure